README.rst

```diff
@@ -21,9 +21,9 @@ discretize
     :target: https://www.codacy.com/app/lindseyheagy/discretize?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=simpeg/discretize&amp;utm_campaign=Badge_Grade
     :alt: codacy status
 
-.. image:: https://www.quantifiedcode.com/api/v1/project/97126041a7e7400fbf1c5555bed514dc/badge.svg
-    :target: https://www.quantifiedcode.com/app/project/97126041a7e7400fbf1c5555bed514dc
-    :alt: Code issues
+.. .. image:: https://www.quantifiedcode.com/api/v1/project/97126041a7e7400fbf1c5555bed514dc/badge.svg
```

Comment:
would you mind deleting this? There is no need to keep it sticking around. Thanks!